mizar/obj/endpoint.py

```diff
@@ -106,7 +110,10 @@ def get_obj_spec(self):
             "hostmac": self.droplet_mac,
             "cnidelay": self.cnidelay,
             "provisiondelay": self.provisiondelay,
-            "pod": self.pod
+            "pod": self.pod,
+            "ingress_networkpolicies": self.ingress_networkpolicies,
+            "egress_networkpolicies": self.egress_networkpolicies,
+            "data_for_networkpolicy": self.data_for_networkpolicy,
```

Comment:
when endpoint first created, there is no ingress_networkpolicices in the spec. did you add three additional fields in spec?

Reply:
Yes, I added in Endpoint.__init__

Reply:
It's not necessary to add this here because when you create the endpoint obj, these fields are not defined. It's always going to be default. You can reference self.bouncers and self.backends.

Reply:
Removed from spec related.